server/crashmanager/migrations/0003_auto_20210429_0908.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.14 on 2021-04-29 09:08
+
+import django.core.files.storage
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('crashmanager', '0002_add_taskmanager'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='testcase',
+            name='test',
+            field=models.FileField(
+                storage=django.core.files.storage.FileSystemStorage(
+                    location='/home/lao/dev/mozilla/FuzzManager/server'
```

Comment:
The current source refers to `/home/decoder` ... so changing to `/home/lao` isn't better, just different. I don't know what the solution is to this problem. Can we reference a setting instead?

Reply:
I tried using [a callable to define storage](https://docs.djangoproject.com/en/dev/topics/files/#using-a-callable) but it's only available in Django 3.1

So i'm using  `settings.BASE_DIR` in the migration.